BunqSdk.Tests/Model/Generated/Endpoint/MonetaryAccountBankTest.cs

```diff
@@ -14,12 +14,12 @@ public class MonetaryAccountBankTest : BunqSdkTestBase
         /// <summary>
         /// Config values
         /// </summary>
-        private const string Status = "CANCELLED";
-        private const string SubStatus = "REDEMPTION_VOLUNTARY";
-        private const string Reason = "OTHER";
-        private const string ReasonDescription = "Because this is a test";
-        private const string Currency = "EUR";
-        private const string Description = "Test C# monetary account";
+        private const string MonetaryAccountBankStatus = "CANCELLED";
+        private const string MonetaryAccountBankSubStatus = "REDEMPTION_VOLUNTARY";
+        private const string MonetaryAccountBankReason = "OTHER";
+        private const string MonetaryAccountBankReasonDescription = "Because this is a test";
+        private const string MonetaryAccountBankCurrency = "EUR";
```

Comment:
Lets make this name future proof, `MonetaryAccountBankCurrencyEur`. If we start supporting other currencies then there will be less names to change 👍